src/server/json_family_test.cc

```diff
@@ -107,19 +107,19 @@ TEST_F(JsonFamilyTest, SetGetFromPhonebook) {
   ASSERT_THAT(resp, "OK");
 
   resp = Run({"JSON.GET", "json", "$.address.*"});
-  EXPECT_EQ(resp, string_view(R"(["New York","NY","21 2nd Street","10021-3100"])", 46));
+  EXPECT_EQ(resp, R"(["New York","NY","21 2nd Street","10021-3100"])");
 
   resp = Run({"JSON.GET", "json", "$.firstName", "$.age", "$.lastName"});
-  EXPECT_EQ(resp, string_view(R"({"$.age":[27],"$.firstName":["John"],"$.lastName":["Smith"]})", 60));
+  EXPECT_EQ(resp, R"({"$.age":[27],"$.firstName":["John"],"$.lastName":["Smith"]})");
 
   resp = Run({"JSON.GET", "json", "$.spouse.*"});
-  EXPECT_EQ(resp, string_view("[]", 2));
+  EXPECT_EQ(resp, "[]");
 
   resp = Run({"JSON.GET", "json", "$.children.*"});
-  EXPECT_EQ(resp, string_view("[]", 2));
+  EXPECT_EQ(resp, "[]");
 
   resp = Run({"JSON.GET", "json", "$..phoneNumbers[1].*"});
-  EXPECT_EQ(resp, string_view(R"(["646 555-4567","office"])", 25));
+  EXPECT_EQ(resp, R"(["646 555-4567","office"])");
```

Comment:
what happens if you call `Run({"JSON.GET", "json", "$.none"}); ` ?
the correct behavior is to return an empty array.